gwctl/cmd/root.go

```diff
@@ -94,3 +96,15 @@ func getParams(path string) *cmdutils.CmdParams {
 
 	return params
 }
+
+func newVersionCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "version",
+		Short: "Print the version information of gwctl",
+		Long:  `Print the version information of gwctl, including version, git commit and build date.`,
+		Run: func(*cobra.Command, []string) {
+			fmt.Println(version.GetVersionInfo())
+		},
+	}
+	return cmd
+}
```

Comment:
nit: Shouldn't this be moved to a separate file like the rest of the subcommands?

Reply:
Sure can! Thanks